task9/Trainings_Pages/Homework.html

```diff
@@ -0,0 +1,252 @@
+        <div class="programs_demanted">Marketing & Sales</div>
+        <div class="programs_demanted">System and network administration</div>
+        <div style="clear:both"></div>
+        <div id="2" class="trainings">upcoming courses</div>
+    <hr>
+<table>
+    <tr>
+        <td>
+            <div class="table">Программирование на GO</div>
+            <div>20.08.2020 - 20.12.2020</div>
+            <div>Занятия вторник и четверг с 18:30 до 20:30</div>
+        </td>
+<td>
+    <div><strong>750 BYN</strong></div>
+        <div>128 уч.ч (три месяца)</div>
+        <div><strong>Осталось</strong> 10 из 20 мест</div>
+        </td>
+<td class="part_table">
+    <a class="colomn">Записаться</a>
+    <a href="#openModalGo" class="colomn2">Программа курса</a>
+<div id="openModalGo" class="modal">
+    <div>
+    <a href="#close" title="Close" class="close">&#10006;</a>
+<h2>Программирование на GO</h2>
+    <hr>
+        <div style="overflow: auto; height: 450px;">
+        <div>Курс “Программирование на языке Go” поможет Вам начать работать с Go,
+        причем с самого начала эф­фективно использовать все его особенности и
+        богатые стандартные библиотеки для написания понятных, идиоматичных и
+        эффективных программ. В рамках курса вы сможете познакомиться с синтаксисом языка Go, основными
+        типами данных, управляющими конструкциями, средствами параллельного
+        программирования (встроенные в язык потоки, синхронизация потоков через
+        каналы и другие средства организации многопоточных программ).
+        Go был задуман в сентябре 2007 года Робертом Грисемером (Robert Griesemer),
+        Робом Пайком (Rob Pike) и Кеном Томпсоном (Ken Thompson) из Google и
+        анонсирован в ноябре 2009 года. Целью разработки было создание
+        выразительного, высокоэффективного как при компиляции, так и при выполнении
+        программ языка программирования, позволяющего легко и просто писать надежные
+        высокоинтеллектуальные программы.</div>
+        <div>Полная программа курса содержит в себе:<br> - Основы языка GO;<br> - Функции;<br> - Обработка исключительных ситуаций;<br> - Структуры и интерфейсы;<br> - Параллельное праграммирование;<br>
+        - Разработка HTTP сервера;<br> - Работа с реляционной базой данных;<br> - Разработка клиентской части;<br> - Деплой и тестирование.</div>
+        </div>
+    </div>
+</div>
+</td>
+</tr>
+</table>
+<table>
+    <tr>
+        <td>
+            <div class="table">JavaScript</div>
+            <div>01.09.2020 - 30.12.2020</div>
+            <div>Занятия вторник и пятница с 18:30 до 21:20</div>
+        </td>
+<td>
+    <div><strong>859 BYN</strong></div>
+    <div>120 уч.ч (три месяца)</div>
+    <div><strong>Осталось</strong> 8 из 12 мест</div>
+</td>
+<td class="part_table">
+    <a class="colomn">Записаться</a>
+    <a href="#openModalJava" class="colomn2">Программа курса</a>
+<div id="openModalJava" class="modal">
```

Comment:
You can move all your modals to the bottom of document.

Reply:
not understand what you want from me((((